platform-sdk/swirlds-config-impl/src/main/java/com/swirlds/config/impl/internal/ConfigDataFactory.java

```diff
@@ -46,7 +52,7 @@ class ConfigDataFactory {
      */
     private final ConverterService converterService;
 
-    ConfigDataFactory(final Configuration configuration, final ConverterService converterService) {
+    ConfigDataFactory(@NonNull final Configuration configuration, @NonNull final ConverterService converterService) {
         this.configuration = CommonUtils.throwArgNull(configuration, "configuration");
```

Comment:
I thought we recently agreed to `Objects.requireNonNull` - perhaps you don't wish to change this existing code?

Reply:
I did this PR before we had that discussion ;)